src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -1116,3 +1118,25 @@ def pd_read_json(file) -> "DataFrame":
 
     with read_open(file) as f:
         return pd.read_json(f, orient="records", lines=True)
+
+
+def get_uuid_by_mac_id():
+    try:
+        node = str(uuid.getnode())
+        if node == "0":
+            raise ValueError("node value is 0")
+        return uuid.uuid3(uuid.NAMESPACE_OID, node)
```

Comment:
we cannot direcctly use mac and user_id as they are PII. Are we following some existing practice here by doing uuid?

Reply:
Also, where do we have some ready-made practices? I can refer to it.

Reply:
done.